compilers/camk.in

```diff
@@ -9,5 +9,5 @@ ifeq ($(PIERNIK_DEBUG), 1)
    F90FLAGS += -ggdb -O0 -ffpe-trap=zero,overflow,invalid -fcheck=all -fno-omit-frame-pointer
    F90FLAGS += -Wall -W -Wextra -pedantic-errors -Wno-unused-function
 else
-   F90FLAGS += -O2 -fstack-arrays
```

Comment:
It used to have a severe performance impact. If you get random segfaults, you can either increase stack size, or identify code that creates too many temp arrays on stack